ngshare/ngshare.py

```diff
@@ -886,6 +886,15 @@ def get(self):
         self.write('\n' + self.request.uri + '\n' + self.request.path + '\n')
 
 
+class HealthCheckHandler(RequestHandler):
+    'Health check handler on /healthz, for k8s'
```

Comment:
As an informal convention, here should be `'/healthz'` and in `__doc__` for `def get(self)` it can have some descriptions.

Reply:
Fixed